intro-HPC/ch_account.tex

```diff
@@ -233,9 +233,9 @@ \subsection{How do SSH keys work?}
 
   You will need to generate a new key pair, when:
   \begin{enumerate}
-    \item  you don't have a key pair yet,
-    \item  you forgot the passphrase protecting your private key,
-    \item  or your private key was compromised.
+    \item  you don't have a key pair yet
```

Comment:
Actually, keep the `,` and `.`

Reply:
I disagree. Either you make a sentence or you make a list. But a list with punctuation is weird, and not proper grammar.